packages/next-swc/crates/next-core/src/next_client/context.rs

```diff
@@ -162,7 +161,6 @@ pub async fn get_client_resolve_options_context(
             Vc::upcast(ModuleFeatureReportResolvePlugin::new(project_path)),
             Vc::upcast(UnsupportedModulesResolvePlugin::new(project_path)),
             Vc::upcast(NextSharedRuntimeResolvePlugin::new(project_path)),
-            Vc::upcast(get_invalid_server_only_resolve_plugin(project_path)),
```

Comment:
That looks unrelated to this change?

Reply:
Yeah you're right... seems like a bad rebase